doc/gromacs.rst

```diff
@@ -108,3 +108,46 @@ The corresponding classes are:
 
     GromacsTopologyFile
     GromacsGroFile
+
+Example usage
+-------------
+
+Many exciting possibilities are available with `GromacsTopologyFile` and `GromacsGroFile`.
+For example, ParmEd is the first software package to enable conversion of Gromacs simulation
+input files to AMBER format in just a few lines of code::
+
+    >>> from chemistry import gromacs, amber
+    >>> gmx_top = GromacsTopologyFile('topol.top')
+    >>> gmx_gro = GromacsGroFile('conf.gro')
+    >>> gmx_top.box = gmx_gro.box # Needed because .prmtop contains box info
+    >>> gmx_top.positions = gmx_gro.positions
+    >>> amb_prm = AmberParm.load_from_structure(gmx_top)
+    >>> amb_prm.write("prmtop")
+    >>> amb_inpcrd = amber.AmberAsciiRestart("inpcrd", mode="w")
+    >>> amb_inpcrd.coordinates = np.array(gmx_top.positions.value_in_unit(u.angstrom)).reshape(-1,3)
```

Comment:
That's something I've been meaning to standardize, although it would be good to solicit input, I guess.

It would be nice if *every* class that carried coordinates made them available in the same way.  My preference would be to call it `coordinates` (currently, `positions` is standardized as the unit-bearing version -- I want a non-unit bearing version as well).